focal/debian/control

```diff
@@ -61,9 +61,9 @@ Section: libdevel
 Depends: libignition-math6-dev (= ${binary:Version}),
          libeigen3-dev,
          ${misc:Depends}
-Replaces: libignition-math-dev (<< ${binary:Version}),
+Replaces: libignition-math-dev (<< 6.0.0~),
```

Comment:
This was set to handle Debian package with is in versions 6.10.0 on jammy. I assume that lintian was complaining, is this correct? Can we leave it as it was or change the 6.0.0 by 6.10.0?

Reply:
This is the focal control file.

Reply:
The warning I got was https://lintian.debian.org/tags/version-substvar-for-external-package

If you look at `libignition-math6-dev`, the replaces uses `6.0.0`, but the `libignition-math6-eigen3-dev` uses ` ${binary:Version}`. Should they be the same?

Reply:
I've removed this change in https://github.com/gazebo-release/gz-math6-release/pull/22/commits/09634d7925b594a987efa94c59ac64da9e1987cd to unblock this PR, but we should probably address the warning at some point.